src/components/ItemEditorPage/RightPanel/RightPanel.tsx

```diff
@@ -265,9 +290,15 @@ export default class RightPanel extends React.PureComponent<Props, State> {
               const isItemLocked = collection && isLocked(collection)
               const canEditItemMetadata = this.canEditItemMetadata(item)
 
-              const actionableCategories = item ? getOverridesCategories(item.contents) : []
+              let actionableCategories = item ? getOverridesCategories(item.contents) : []
               const wearableCategories = item ? getWearableCategories(item.contents) : []
 
+              if (data?.category === WearableCategory.SKIN) {
+                actionableCategories = actionableCategories.filter(actionableCategory =>
+                  !getSkinHiddenCategories().includes(actionableCategory) && actionableCategory !== WearableCategory.SKIN
+                )
+              }
```

Comment:
What if we give the `getOverridesCategories` the item's category as well and put this section of code there?

Reply:
done ✅